specs/permissions.md

```diff
@@ -137,15 +115,18 @@ The scheme object has members that describe the permission within the context of
         "adminDescription": "Allows the app to read and report the signed-in user's activity in the app.",
         "userConsentDisplayName": "Read and write app activity to users'activity feed",
         "userConsentDescription": "Allows the app to read and report the signed-in user's activity in the app.",
-        "requiresAdminConsent": true
+        "requiresAdminConsent": true,
+        "privilegeLevel": 3
```

Comment:
Using integer values is problematic because how does anyone know if 1 is high privilege or 5 is?